packages/deparser/src/deparser.js

```diff
@@ -2733,7 +2791,7 @@ export default class Deparser {
     );
     output.push(`${NEWLINE_CHAR})`);
 
-    if (relpersistence === 'p' && node.hasOwnProperty('inhRelations')) {
+    if (node.hasOwnProperty('inhRelations')) {
```

Comment:
I couldn't find any reason why this condition needs to be here. It looks like temp tables can INHERIT too